util/testleak/leaktest.go

```diff
@@ -33,6 +33,7 @@ func interestingGoroutines() (gs []string) {
 	buf = buf[:runtime.Stack(buf, true)]
 	ignoreList := []string{
 		"created by github.com/pingcap/tidb.init",
+		"gopkg.in/natefinch/lumberjack%2ev2.(*Logger).millRun",
```

Comment:
duplicated as https://github.com/pingcap/tidb/pull/20935 to fix https://github.com/pingcap/tidb/issues/20845